src/schedulers/schedulers.module.ts

```diff
@@ -0,0 +1,22 @@
+import { Module } from '@nestjs/common';
```

Comment:
I don't think you need another schedulers module defined here, this needs to be refactored. `ElasticsearchModule`s are re-imported every time, but the general authentication used is the same.

This needs to be changed, but we can do it later.

Reply:
okay.